roles/wordpress-setup/defaults/main.yml

```diff
@@ -2,6 +2,20 @@ site_uses_local_db: "{{ site_env.db_host == 'localhost' }}"
 nginx_wordpress_site_conf: wordpress-site.conf.j2
 nginx_ssl_path: "{{ nginx_path }}/ssl"
 
+ssl_default_site:
+  no_default:
+    site_hosts:
+      - canonical: example.com
```

Comment:
It's a little odd to hardcode this is a 2nd place especially since it's the production value only. Is this actually needed?

Reply:
(comment was addressed in internal discussion)